src/kvstore/plugins/hbase/hbase.thrift

```diff
@@ -398,7 +398,7 @@ service THBaseService {
     4: binary qualifier,
 
     /** the expected value, if not provided the
-        check is for the non-existence of the
+        check is for the nonexistentce of the
```

Comment:
Sigh, this isn't the current version of my branch... I seem to have lost some fixes in my iterations...